src/vue/modules/verification/general-form/store/wrap_document.js

```diff
@@ -0,0 +1,8 @@
+import { DocumentContainer } from '@/js/helpers/DocumentContainer'
+
+export function wrapDocument (document, type) {
```

Comment:
useless wrapper definition, see no reason why you cannot use `new DocumentContainer(...)` directly

Reply:
@max-nes just to make code cleaner, it looked like a mess, because

```
  new DocumentContainer({
     type: DOCUMENT_TYPES.documentType,
     documents[DOCUMENT_TYPES.DOCUMENT_TYPE]
  })
```

looked very bad when repeated a lot of times.

Reply:
ok, if we keep the file, pls make the file be kebab-cased